src/BlazorUI/Bit.BlazorUI/Components/Utilities/Icon/BitIcon.scss

```
@@ -134,3 +138,208 @@
+    36% { transform: rotate(12deg); }
+    40%, 100% { transform: rotate(0deg); }
+}
+
+
+// An icon given a click handler is announced and operated as a button, so it also has to look and
+// focus like one. It keeps the icon's own shape - the ring follows the border radius of the variant
+// it is drawn in - and a disabled one stops answering the pointer entirely.
+.bit-ico-int {
+    cursor: pointer;
+    position: relative;
+    // A double click on a control should not leave the glyph - or, for a ligature-based set, the word
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)